nbproject/project.properties

```diff
@@ -1,7 +1,9 @@
+browser.autorefresh.Chrome.INTEGRATED=true
```

Comment:
Could we add the /nbproject folder to the gitignore? not sure who else is using netbeans but it would make it a little cleaner.